capstone/src/main/java/com/google/step/data/RestaurantScore.java

```diff
@@ -0,0 +1,46 @@
+package com.google.step.data;
+
+/**
+ * Class that holds the score for a specific restaurant.
+ */
+public class RestaurantScore {
+
+    // The name should always remain the same but the score will change when calculating it.
+    private final String restaurantKey;
```

Comment:
I believe this field should be a long

https://github.com/googleinterns/step93-2020/blob/master/capstone/src/main/java/com/google/step/data/Restaurant.java#L26

Reply:
Fixed.